modules/server/src/main/scala/navigate/server/tcs/TcsChannels.scala

```diff
@@ -493,6 +509,40 @@ object TcsChannels {
     )
   }
 
+  object GuiderGains {
+    def build[F[_]](
+      service: EpicsService[F],
+      top:     String
+    ): Resource[F, GuiderGainsChannels[F]] =
+      for {
+        p1tipGain   <- service.getChannel[String]("pwfs1:dc:detSigInitFgGain.A")
+        p1tiltGain  <- service.getChannel[String]("pwfs1:dc:detSigInitFgGain.B")
+        p1FocusGain <- service.getChannel[String]("pwfs1:dc:detSigInitFgGain.C")
+        p1Reset     <- service.getChannel[BinaryYesNo]("pwfs1:dc:initSigInit.J")
+        p2tipGain   <- service.getChannel[String]("pwfs2:dc:detSigInitFgGain.A")
+        p2tiltGain  <- service.getChannel[String]("pwfs2:dc:detSigInitFgGain.B")
+        p2FocusGain <- service.getChannel[String]("pwfs2:dc:detSigInitFgGain.C")
+        p2Reset     <- service.getChannel[BinaryYesNo]("pwfs2:dc:initSigInit.J")
+        oitipGain   <- service.getChannel[String]("oiwfs:dc:detSigInitFgGain.A")
+        oitiltGain  <- service.getChannel[String]("oiwfs:dc:detSigInitFgGain.B")
+        oiFocusGain <- service.getChannel[String]("oiwfs:dc:detSigInitFgGain.C")
+        oiReset     <- service.getChannel[BinaryYesNo]("oiwfs:dc:initSigInit.J")
```

Comment:
Top should be a parameter for every system: PWFS1, PWFS2, OIWFS. That helps when testing with an EPICS simulator